test/src/api/protocol/testSetValue.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+const Globals = require('../../../lib/globals.js');
+
+describe('setValue command', function () {
+  before(function () {
+    Globals.protocolBefore();
+  });
+
+  it('should clear value and setValue', function () {
+    return Globals.protocolTest({
+      commandName: 'setValue',
+      args: [
+        'weblogin',
+        'nightwatch',
+        function (result) {
+          assert.deepStrictEqual(result.value, null);
+          assert.deepStrictEqual(result.status, 0);
+        }
+      ],
+      assertion: function (opts) {
+        assert.notStrictEqual(-1, ['findElements', 'clearElement', 'sendKeysToElement'].indexOf(opts.command));
```

Comment:
not very clear what should happen here. what are we testing?

Reply:
We wanted to tests, that `clearElement` is called before we do `sendKeysToElement`

Reply:
And this only checks if opts.command is either one of those items from the array.

Reply:
yes, it checks if all these commands are called or not, as if one of the command is missing than, it will throw an error

Reply:
So the assertion is called multiple times? You can use something like `assert.ok([].contains())` instead.

Reply:
yes, it's called multiple times

Reply:
```suggestion
        assert.ok(['findElements', 'clearElement', 'sendKeysToElement'].contains(opts.command));
```

Reply:
yup, changed this in recent commit 😅

Reply:
Also, `Array` don't have `.contains` method, instead we use `[].includes`

Reply:
Ah yes, sorry.